index.html

```diff
@@ -4,28 +4,363 @@
+            <div class="socials">
+              <a href="//linkedin.com/in/michaelhumiston" target="_blank" rel="noopener noreferrer">
+                <img src="img/icons/socials-black.svg#linkedin" />
+              </a>
+              <a href="//github.com/mhum" target="_blank" rel="noopener noreferrer">
+                <img src="img/icons/socials-black.svg#github" />
+              </a>
+            </div>
+            <h4>Michael <strong>Humiston</strong></h4>
+            <h5><em>Partner</em></h5>
+          </div>
+          <div class="person card">
+            <figure>
+              <img src="img/team/becky-color.png" />
+            </figure>
+            <div class="socials">
+              <a href="//twitter.com/BeckyPollard_" target="_blank" rel="noopener noreferrer">
+                <img src="img/icons/socials-black.svg#twitter" />
+              </a>
+              <a href="//github.com/BeckyPollard" target="_blank" rel="noopener noreferrer">
+                <img src="img/icons/socials-black.svg#github" />
+              </a>
+            </div>
+            <h4>Becky <strong>Pollard</strong></h4>
+            <h5><em>Developer</em></h5>
+          </div>
+          <div class="person card">
+            <figure>
+              <img src="img/team/siep-color.png" />
+            </figure>
+            <div class="socials">
+              <a href="//twitter.com/ikbensiep" target="_blank" rel="noopener noreferrer">
+                <img src="img/icons/socials-black.svg#twitter" />
+              </a>
+              <a href="//github.com/ikbensiep" target="_blank" rel="noopener noreferrer">
+                <img src="img/icons/socials-black.svg#github" />
+              </a>
+            </div>
+            <h4>Simon <strong>Hoekstra</strong></h4>
+            <h5><em>Developer</em></h5>
+          </div>
+        </div>
+      </section>
+      
+      <!-- projects -->
+      <section id="projects">
+        <h1>Our <strong>clients</strong></h1>
+        <ul class="flex-row clients">
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/embreate-logo.svg" alt="Embreate logo"></a>
+            <small>Embreate</small>
+          </li>
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/orderup-logo-white.svg" alt="OrderUp logo"></a>
+            <small>OrderUp</small>
+          </li>
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/toronto-logo.svg" alt="City of Toronto logo"></a>
+            <small>City of Toronto</small>
+          </li>
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/underknown-logo-white.svg" alt="Underknown logo"></a>
+            <small>Underknown</small>
+          </li>
+          <li>
+            <a href="" target="_blank" rel="noopener"><img src="img/clients/svg/zerolens-white.svg" alt="Zerolens logo"></a>
```

Comment:
Zerolens has been renamed to 'Colorful', so we should probably use that: https://www.colorful.app/